README.md

```diff
@@ -1,23 +1,96 @@
+
+# Action Module
+
+The purpose of the Action module is to isolate the ScanCore decision logic.
+
+## Overview
+
+![Overview of the Action module](./assets/scancore-decision-making.png)
+
+With the Action module, ScanCore's decision making process has been modified to:
+
+1. ScanCore makes a decision through `decide()` in the Action module
+2. `decide()` puts the decision into `public.decisions` inside ScanCore's database; the decisions table is essentially a queue
+3. The inserted decision gets replicated to `history.decisions`
+4. When ScanCore reaches the end of a scan, it executes all decisions in `public.decisions`
+5. Executed decisions are removed from `public.decisions`
+
+## Allowing External Decisions
+
+![Overview of the Action module working with the AI](./assets/scancore-ai-decision-making.png)
+
+Two additional pieces have been added along with the Action module to allow the AI to make decisions:
+
+### 1. `call_action-decide` CLI tool
+
+`call_action-decide` calls `decide()` in the Action module to insert external decisions. It takes 2 arguments in the form of switches:
+
+1. (required) `--decision`, and
+2. (optional) `--node-uuid`
+
+Since this tool exists on all nodes and strikers, it is unnecessary to add additional arguments, i.e., host UUID, to determine the script's environment. Only decisions 10 and 11 (refer to the [striker decisions table](#striker-decisions)) require `--node-uuid` because it specifies which node to boot/reboot.
+
+Example usage:
+
+```
+call_action-decide --decision "10" --node-uuid "f5ac6b0b-9ef2-4b52-abcb-8fd46135a65c"
+```
+
+### 2. `scancore::prevent_decision_making` flag in `/etc/striker/striker.conf`
+
+The `prevent_decision_making` flag prevents ScanCore from queuing decisions. It can be set to:
+
+- `0`: allow ScanCore to queue decisions, or
+- `1`: prevent ScanCore from queuing decisions
+
+## Decisions Definition
+
+### Node Decisions
+
+These decisions are only made by **nodes**.
+
+| Number | Definition                                                                                                                                                |
+| ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1      | Enter (or reenter) stable **power** state and clear power Warnings.                                                                                       |
+| 2      | Enter (or reenter) stable **temperature** state and clear temperature Warnings.                                                                           |
+| 3      | Enter **power** Warning state and send notifications stating that power abnormalities were detected.                                                      |
```

Comment:
I think the `w` in `Warning` should be small case.

Reply:
Changed as suggested.